src/vm/method.cpp

```diff
@@ -1213,16 +1213,14 @@ COR_ILMETHOD* MethodDesc::GetILHeader(BOOL fAllowOverrides /*=FALSE*/)
 }
 
 //*******************************************************************************
-MetaSig::RETURNTYPE MethodDesc::ReturnsObject(
-#ifdef _DEBUG
-    bool supportStringConstructors,
-#endif
-    MethodTable** pMT
-    )
+ReturnKind MethodDesc::ReturnsObject(INDEBUG(bool supportStringConstructors))
 {
     CONTRACTL
     {
-        if (FORBIDGC_LOADER_USE_ENABLED()) NOTHROW; else THROWS;
+        if (FORBIDGC_LOADER_USE_ENABLED()) 
```

Comment:
This CONTRACT is pretty common and it is always on a single line. Keep it that way here as well.

Reply:
Fixed, thanks.